src/util/psbt/mod.rs

```diff
@@ -210,6 +213,234 @@ impl PartiallySignedTransaction {
 
         Ok(())
     }
+
+    /// Signs the input at `input_index` with private key `sk`.
+    ///
+    /// # Errors
+    /// If an attempt is made to sign a taproot input.
+    pub fn sign<C: Signing>(&mut self, sk: &PrivateKey, input_index: usize, secp: &Secp256k1<C>) -> Result<(), SighashError> {
```

Comment:
Perhaps clearly disambiguate with `bitcoin::PublicKey` here.

Reply:
This is `rust-bitcoin` so naked key implies, to me at least, `bitcoin::PrivateKey`. (I get that in miniscript it makes sense to disambiguate the keys.)